tests/org.jboss.tools.batch.ui.bot.test/src/org/jboss/tools/batch/ui/bot/test/AbstractBatchTest.java

```diff
@@ -109,15 +104,11 @@ private static void importProject() {
 	 */
 	protected static void removeProject(Logger log) {
 		log.info("Removing " + PROJECT_NAME);
-		IProject project = ResourcesPlugin.getPlugin().getWorkspace().getRoot().getProject(PROJECT_NAME);
 		try {
-			project.delete(true, true, null);
-		} catch (CoreException coreExc) {
-			log.error("Could not delete project or its content");
-			throw new RedDeerException("Project delete operation was not possible to process");
-		} catch (OperationCanceledException cancelExc) {
-			log.error("Delete operation was canceled");
-			throw new RedDeerException("Project delete operation was canceled");
+			org.jboss.reddeer.direct.project.Project.delete(PROJECT_NAME, true, true);
+		} catch (RuntimeException runtimeExc) {
+			log.info("Runtime exception appeared, re-trying deleting of project");
+			org.jboss.reddeer.direct.project.Project.delete(PROJECT_NAME, true, true);
```

Comment:
Hm, seems to be working anyway. I believe, it is because of catching of exception during process of deleting a project and repeatable call of delete operation that erases the rest of the project, isn't it?